tests/python/contrib/test_tensorrt.py

```diff
@@ -874,13 +960,82 @@ def test_densenet121():
     run_and_verify_model("densenet121")
 
 
+def test_tensorrt_integration():
+    # Integration tests
+    test_alexnet()
+    test_resnet18_v1()
+    test_resnet18_v2()
+    test_squeezenet()
+    test_mobilenet()
+    test_mobilenet_v2()
+    test_vgg11()
+    test_densenet121()
+
+
+def test_dynamic_offload(data_shape=(1, 32, 8, 8), k_shape=(1, 32, 3, 3)):
```

Comment:
Lets move the args to variables inside the test. Same for the serialization tests